.github/workflows/model_ci.yml

```diff
@@ -68,7 +68,7 @@ jobs:
 
     - name: Post cross-val results to PR
       if: steps.cvnlu.outcome == 'success'
-      uses: amn41/comment-on-pr@comment-file-contents
+      uses: samsucik/comment-on-pr@comment-file-contents
```

Comment:
Just curious, why does this depend on your username?

Reply:
@kedz see my comment [here](https://www.notion.so/Re-deploying-Carbon-bot-8de15299e3fa4184be8fe9be17830b3d?d=032a248c75354962bf519bb4d67013e6#d07ab3990e51498689da123cec92fd87). This warrants a follow-up issue to consolidate and move the relevant code over to some Rasa repo.